src/types.ts

```diff
@@ -0,0 +1,70 @@
+export enum FlexPaymentActions {
+  addMinOrFixed = 1,
+  addMaxOrFixed = 2,
+  addSubRadio = 3,
+  addSubRadioWithExtraCost = 4,
+  addSubCheckbox = 5,
+  addSubCheckboxWithExtraCost = 6,
+  setTip = 7,
+  externalPricing = 8,
+  userEntry = 9,
+  setTotal = 10,
+}
+
+export type PaymentSdkConfigType = {
+  campaignId: string;
+  formId?: string;
+  deviceId?: string;
+  xcoobeeDeviceId?: string;
+  source?: string;
+};
+
+export type LogicReferenceOptionType = string | [string, number];
+
+export type LogicReferenceType = string | SimpleListType | ListWithCostType;
+
+export type FlexLogicType = {
+  a: FlexPaymentActions;
+  m?: number;
+  M?: number;
+  o?: number;
+  r?: LogicReferenceType;
+};
+
+export type FlexItemType = {
+  "0-3"?: number;
+  "0-5"?: number;
+  "0-6"?: string;
+  l?: FlexLogicType | FlexLogicType[];
+};
+
+export type FlexItemHumanizedType = DefaultPayUrlConfigType & {
+  logic?: FlexLogicType | FlexLogicType[];
+};
+
+export type SimpleListType = string[];
+export type ListWithCostType = [string, number][];
+
+export interface Base64Interface {
+  atob(base64: string): string;
```

Comment:
Looks like `atob` is not being used but `btoa` is being used. We can remove `atob`. But if you use the `abab` package, then we wont' need Base64Interface.